src/hocs/FilterNav.js

```diff
@@ -0,0 +1,61 @@
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { curry } from 'lodash';
+import { actions as bountiesActions } from 'public-modules/Bounties';
+import {
+  rootBountiesSelector,
+  bountiesCategoryFiltersSelector,
+  bountiesPlatformFiltersSelector,
+  bountiesSortFilterSelector
+} from 'public-modules/Bounties/selectors';
+import { categoriesSelector } from 'public-modules/Categories/selectors';
+
+function FilterNavComponentHOC(WrappedComponent, config) {
+  let mapStateToProps = null;
+  let mapDispatchToProps = null;
+
+  switch (config.type) {
+    case 'explorer':
```

Comment:
why even do this? Everything just rolls down anyways?

Reply:
i had big plans for this but then realized it was not necessary for the platform filter on the leaderboard